webapp/src/widgets/tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import './tooltip.scss'
+
+type Props = {
+    tooltip: string
```

Comment:
Let's rename this to `title`
```suggestion
    title: string
```

Reply:
Sure thing